### ISSUE TITLE
[HOLD for payment 2023-06-02] [$1000] Attachment modal shortcut bugs on native

### ISSUE THREAD

**Issue by luacmartins** [Expensify/App#18268]
Coming from [this comment](https://github.com/Expensify/App/issues/17499#issuecomment-1529483118), we'll address:

Issue 3 - Attachment modal

iOS: Add attachment modal does not navigate based on arrow keys
iOS | Android: Left & Right arrow keys does not work (seems to be duplicate of the issue above)
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01da468373ed9db4e0</li>
        <li>Upwork Job ID: 1653407518037057536</li>
        <li>Last Price Increase: 2023-05-02</li>
    </ul>
</details>

**Comment by MelvinBot** [comment]
Triggered auto assignment to @twisterdotcom (`Bug`), see https://stackoverflow.com/c/expensify/questions/14418 for more details.

**Comment by MelvinBot** [comment]
Job added to Upwork: https://www.upwork.com/jobs/~01da468373ed9db4e0

**Comment by MelvinBot** [comment]
### Bug0 Triage Checklist ([Main S/O](https://stackoverflowteams.com/c/expensify/questions/14418))

- [ ] This "bug" occurs on a supported platform (ensure `Platforms` in OP are ✅)
- [ ] This bug is not a duplicate report (check [E/App issues](https://github.com/Expensify/App/issues) and [#expensify-bugs](https://expensify.slack.com/archives/C049HHMV9SM))
    - If it is, comment with a link to the original report, close the issue and add any novel details to the original issue instead
- [ ] This bug is reproducible using the reproduction steps in the OP. [S/O](https://stackoverflowteams.com/c/expensify/questions/14418#:~:text=The%20bug%20is%20reproducible)
    - If the reproduction steps are clear and you're unable to reproduce the bug, check with the reporter and QA first, then close the issue.
    - If the reproduction steps aren't clear and you determine the correct steps, please update the OP.
- [ ] This issue is filled out as thoroughly and clearly as possible
    - Pay special attention to the title, results, platforms where the bug occurs, and if the bug happens on staging/production.
- [ ] I have reviewed and subscribed to the linked Slack conversation to ensure Slack/Github stay in sync

**Comment by MelvinBot** [comment]
Triggered auto assignment to @abekkala (`External`), see https://stackoverflow.com/c/expensify/questions/8582 for more details.

**Comment by luacmartins** [comment]
@azimgd please comment on this issue so I can assign you

**Comment by MelvinBot** [comment]
Triggered auto assignment to Contributor-plus team member for initial proposal review - @rushatgabhane (`External`)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @jasperhuangg (`External`), see https://stackoverflow.com/c/expensify/questions/7972 for more details.

**Comment by MelvinBot** [comment]
📣 @rushatgabhane You have been assigned to this job by @luacmartins!
Please apply to this job in Upwork and leave a comment on the Github issue letting us know when we can expect a PR to be ready for review 🧑‍💻
Keep in mind: [Code of Conduct](https://github.com/Expensify/App/blob/main/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md) 📖

**Comment by twisterdotcom** [comment]
Ah, I love the ones where I'm just a money man. Keep me posted!

**Comment by MelvinBot** [comment]
📣 @azimgd You have been assigned to this job by @twisterdotcom!
Please apply to this job in Upwork and leave a comment on the Github issue letting us know when we can expect a PR to be ready for review 🧑‍💻
Keep in mind: [Code of Conduct](https://github.com/Expensify/App/blob/main/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md) 📖

**Comment by azimgd** [comment]
I was able to reproduce "Add attachment modal does not navigate based on arrow keys".

@rushatgabhane Could you connect a hardware keyboard and try to reproduce both, on production Android please.
@luacmartins What should "Left & Right arrow keys" actually do?

**Comment by luacmartins** [comment]
@azimgd left & right should cycle through the attachment list, same as pressing the visual arrows on the screen.

**Comment by azimgd** [comment]
vice versa, the question is "why somebody would use left/right keys to move up/down". I don't see a point and that was submitted as a bug. I'm in favor of just removing this from the scope.

**Comment by twisterdotcom** [comment]
I'm confused about where we're at with this now. We agreed here this needs to be done: https://expensify.slack.com/archives/C01GTK53T8Q/p1683208083045299.

Are we just waiting on the PR to be ready?

**Comment by twisterdotcom** [comment]
Latest update is from 4 hours ago in the PR: https://github.com/Expensify/App/pull/18407#issuecomment-1550147983

**Comment by twisterdotcom** [comment]
Deployed to staging 10 hours ago: https://github.com/Expensify/App/pull/18407#issuecomment-1562134209

**Comment by twisterdotcom** [comment]
@rushatgabhane which Upwork job? I'm only seeing this one which is closed: https://www.upwork.com/ab/applicants/1653543158120652800/job-details

**Comment by azimgd** [comment]
@twisterdotcom what's the correct upwork url?

**Comment by twisterdotcom** [comment]
The automated one was this one, but it looks like that was closed for some reason: https://www.upwork.com/jobs/~01da468373ed9db4e0.

Ideally, I'd have found the one @rushatgabhane applied to, but whatever, I've just sent you both an offer to a new job referencing this issue.

Am I right in thinking there's no bonus on this one? We haven't posted a comment like this automatically: https://github.com/Expensify/App/issues/18300#issuecomment-1561748636

**Comment by twisterdotcom** [comment]
Paid out.
